resources/popup.html

```diff
@@ -39,7 +40,7 @@
                 </a>
             </div>
             <div class="col s6">
-                <a class="btn waves-effect lime waves-light tooltipped" href="../analytics/index.html" target="_blank"
+                <a id="analytics-link" class="btn waves-effect lime waves-light tooltipped" target="_blank"
```

Comment:
`target` is set to `_blank` both here and in the `main.js` file. Is it really necessary to open a new window when you are in the options section of Octopeer?